packages/validator/src/commands/convertCmd.ts

```diff
@@ -9,27 +10,52 @@ import { invalidConversionMessage } from '../utils/messages';
 const oas2ois = 'OAS2OIS.json';
 const ois2config = 'OIS2Config.json';
 
-if (process.env.npm_config_template) {
-  console.log(
-    JSON.stringify(convert(process.env.npm_config_specs || process.argv[3], process.env.npm_config_template), null, 2)
-  );
+const args = yargs(hideBin(process.argv))
+  .option('from', {
+    description: 'Name of the source airnode specification format',
+    demandOption: 'to',
+    default: '',
+    string: true,
+  })
+  .option('to', {
+    description: 'Name of the target airnode specification format',
+    demandOption: 'from',
+    default: '',
+    string: true,
+  })
```

Comment:
You can use [`implies`](https://github.com/yargs/yargs/blob/master/docs/api.md#impliesx-y) to require an option A if an option B is provided. I think that would work well for `from` and `to`.